### ISSUE TITLE
Keep on the good work with diversity and education programs

### ISSUE THREAD

**Issue by nanuxbe** [pybee/paying-the-piper#49]
This is clearly not a short-term solution, rather a long-term one but coming up in different threads on this repo, including #14. It seems to me that the funding problem is tightly linked to an awareness problem.

It would emerge that one of the reasons people outside the developer community are not more concerned about funding OSS is because:
- they are not aware what OSS is (ie: people selling gimp on e-bay
- they are not aware how important security in OSS is (large companies want to keep SHA-1 SSL certificates)
- they think creating software is "easy" as in they don't understand the work involved to create quality software (as in your manager doesn't think your team has time to write unit tests or deal with technical debt)

In the light of these facts, I think initiatives like Django Girls, Rails Girls or the BBC micro:bit are good for plenty of reason but are also necessary to educate more and more people to what software actually is and that it takes time and effort to create thins like the game that you just bought for $1 on the app-store or a website.
Since RMS was mentioned in another threat, let me just say that, at the time, it required a lot more knowledge to use a computer, nowadays, a 2 yo can use a tablet, therefore, outside our community, our work doesn't seem as valuable as it used to.

TL;DR; OSS won't get funding from people who don't know how much hard work is being accomplished => let's educate as many people as we can
